ejabberd.te

```diff
@@ -0,0 +1,57 @@
+policy_module(ejabberd,0.0)
+
+
+# Private type declarations
+type ejabberd_t;
+type ejabberd_exec_t;
+type ejabberd_unit_t;
+systemd_unit_file(ejabberd_unit_t)
+type ejabberd_var_lib_t;
+files_type(ejabberd_var_lib_t)
+type ejabberd_var_log_t;
+
+
+init_daemon_domain(ejabberd_t, ejabberd_exec_t)
+
+logging_log_file(ejabberd_var_log_t)
+
+
+# What will we allow
+allow ejabberd_t self:tcp_socket { accept bind connect create getattr getopt listen read setopt write };
+allow ejabberd_t self:udp_socket { bind connect create getattr getopt read setopt write };
+allow ejabberd_t self:unix_dgram_socket { connect create getopt setopt write };
```

Comment:
Please review the existing permission set macros to see if they can be used instead of the verbose permission lists here.